examples/boltzmann_wealth_model_network/README.md

```diff
@@ -0,0 +1,47 @@
+# Boltzmann Wealth Model with Network
+
+## Summary
+
+This is the same Boltzmann Wealth Model, but with a network grid implementation.
```

Comment:
@cauemello (this assumes that you implement the other recommendation about nodes as space) can you add one line that says that agents must be located on a node, there is a limit of one agent on a node, and agents must be connected to other agents in order to trade? ... then continue w/ the generic description below... if that makes sense.